src/types/mod.rs

```diff
@@ -207,9 +207,15 @@ macro_rules! pyobject_native_type_info(
 
             $(
                 #[inline]
-                fn is_type_of(ptr: &$crate::PyAny) -> bool {
+                fn is_type_of(obj: &$crate::PyAny) -> bool {
                     #[allow(unused_unsafe)]
-                    unsafe { $checkfunction(ptr.as_ptr()) > 0 }
+                    unsafe { $checkfunction(obj.as_ptr()) > 0 }
+                }
```

Comment:
Could we implement this in terms of `is_type_of_bound` (or maybe dont need it at all, if thats the default)?

Reply:
I think we can just use the new default here 👍